cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,59 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
```

Comment:
The task states that the test flow should be split into 3 tests, but only 1 test is present in the code.

Reply:
Done